prow/plugins/project/project_test.go

```diff
@@ -303,11 +303,12 @@ func TestProjectCommand(t *testing.T) {
 		fakeClient.Column = tc.previousColumn
 
 		e := &github.GenericCommentEvent{
-			Action: tc.action,
-			Body:   tc.body,
-			Number: 1,
-			Repo:   github.Repo{Owner: github.User{Login: tc.org}, Name: tc.repo},
-			User:   github.User{Login: tc.commenter},
+			Action:       tc.action,
+			Body:         tc.body,
+			Number:       1,
+			IssueHTMLURL: "1",
```

Comment:
Why is this valid?

Reply:
Original implementation used Issue number, and that would not work. We cannot fetch a card with just issue number.